README.rst

```diff
@@ -219,27 +205,27 @@ If you'd like to modify a component via a branch and point to that
 branch in your CESM sandbox, use the following procedure (again, using
 CAM as an example):
 
-#. Create a CAM branch. Since CAM originates from a subversion
-   repository, you will first need to create a branch in that
-   repository. Let's assume you have created this branch and called it
+#. Create a CAM branch. Let's assume you have created this branch and called it
    **my_branch**.
 
-#. Update **Externals.cfg** to point to your branch. You can replace the
-   **tag** entry with a **branch** entry, as follows::
+#. Update **.gitmodules** to point to a hash on your branch. You can replace the
+   **tag** entry with a **hash** entry, as follows, note that we have also changed the url to
+   point to a personal fork::
+
+  [submodule "cam"]
+  path = components/cam
+  url = https://www.github.com/mycamfork/CAM
+  fxDONOTUSEurl = https://www.github.com/ESCOMP/CAM
+  fxtag = 94eaf83
+  fxrequired = ToplevelRequired
```

Comment:
This needs to be indented so it is treated as a code block, right now it's a mangled quote:

![modifying component](https://github.com/user-attachments/assets/8dcfb5cf-72e1-47ca-b4b9-fadbab069332)

Everything else looks great, thanks for updating this!